js/src/ui/SelectionList/selectionList.css

```diff
@@ -22,6 +22,7 @@
   height: 100%;
   position: relative;
   width: 100%;
+  max-width: 100%;
```

Comment:
This fixes display of accounts on firefox (especially in Parity Bar when selecting default account, the accounts were overlapping).